src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+    { age ? (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+    ) : <></> }
+    { isMarried && sex === 'm' ? (
+      <p className="Person__partner">
+        {partnerName}
+        {' '}
+        is my wife
+      </p>
+    ) : <></> }
```

Comment:
No need for that, it doesn't do anything. Use short circuiting `&&`

Also move this logic outside of template to the variable and just render it here.
```
const husbandWife = `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`;
const spouse = isMarried ? husbandWife : 'I am not married';
```
and use it in the template like `{spouse}`